point-of-sale-app/inventory/src/main/java/com/google/abmedge/inventory/dto/Item.java

```diff
@@ -0,0 +1,135 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.abmedge.inventory.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+public class Item {
+  private UUID id;
+  private String name;
+  private String type;
+  private Number price;
+  private String imageUrl;
+  private long quantity;
+  private List<String> labels;
+
+  public Item() { }
```

Comment:
I'm not a java expert, but why do you have two different constructors for `Item`?

Reply:
I only had the second one _(the one taking in 2 arguments)_ because I thought an item should always have an id and a type. Then I had to add the first one _(empty default constructor)_ because it is required by the Gson library when an object via the wire is to be deserialized and to serialize before going in the wire. Gson expects an empty constructer. 

However now looking at the code, the second constructor seems to be used only inside the same class. So I removed it and just kept the other one in 272902463e20a19857a2a27448d40abae0139c27